docs/source/quickstart.mdx

```diff
@@ -33,18 +47,120 @@ Start by installing 🤗 Datasets:
 pip install datasets
 ```
 
-To work with audio datasets, install the [`Audio`] feature:
+Besides 🤗 Datasets, make sure your preferred machine learning framework is installed, so you can convert a dataset to be compatible with a specific framework.
```

Comment:
Nit:

```suggestion
Besides 🤗 Datasets, make sure your preferred machine learning framework is installed:
```

Also, let's put this part after the `pip install datasets[audio]` and `pip install datasets[image]` blocks